test/functional/test-integration.js

```diff
@@ -34,43 +34,33 @@ describe('3p integration.js', () => {
   });
 
   it('should register integrations', () => {
-    expect(registrations).to.include.key('a9');
-    expect(registrations).to.include.key('adblade');
-    expect(registrations).to.include.key('adition');
-    expect(registrations).to.include.key('adform');
+    // Get all ad extensions from test env.
+    const extensions = window.ampTestRuntimeConfig.adExtensions;
```

Comment:
im honestly slightly conflicted with this, as much as its a nicety i'd rather keep tests as stupid simple as possible even with the repetition.